src/lib/vrf_evaluator/vrf_evaluator.ml

```diff
@@ -151,7 +152,7 @@ module Worker_state = struct
         let%bind () =
           Interruptible.lift Deferred.unit (Ivar.read interrupt_ivar)
         in
-        let module Slot = Mina_numbers.Global_slot in
+        let module Slot = Mina_numbers.Global_slot_since_genesis in
```

Comment:
Same as the above comment for `Slot`

Reply:
`Slot` is in scope, so I just removed this.